#!/usr/bin/env python
PACKAGE = "calibrate_odom"

from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()

gen.add("test_distance", double_t, 0, "test_distance", 1.0, 0, 5.0)
gen.add("actual_distance", double_t, 0, "actual_distance", 0, 0, 5.0)
gen.add("odom_scale", double_t, 0, "odom_scale", 1.0, 0, 3.0)
gen.add("start_test", bool_t, 0, "start_test", False)

exit(gen.generate(PACKAGE, "calibrate_odom", "calibrate"))
